hibernate-core/src/main/antlr/sql-script.g

```diff
@@ -57,16 +57,23 @@ options {
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 script
-	: (newLineToSkip)* (statement)* EOF
+	: (newLineToSkip | SPACE | TAB )* (statement)* EOF
     ;
 
 statement
-	: { statementStarted(); } (statementPart (afterStatementPartNewline)*)* DELIMITER (newLineToSkip)* { statementEnded(); }
+	: { statementStarted(); } statementFirstPart (statementPart (afterStatementPartNewline)*)* DELIMITER (SPACE | TAB | newLineToSkip)* { statementEnded(); }
+	;
```

Comment:
maybe you could reorganize the "whitespace" into its own group to make things simpler?

```
script
	: ignoredWhitespace (statement ignoredWhitespace)* EOF
    ;

statement
	: { statementStarted(); } statementFirstPart (statementPart (afterStatementPartNewline)*)* DELIMITER { statementEnded(); }
	;

ignoredWhitespace
	: (newLineToSkip | SPACE | TAB )*
	;
```

Reply:
Thanks @Sanne I made the suggested change.